src/main/java/net/snowflake/client/jdbc/SnowflakeBaseResultSet.java

```diff
@@ -828,9 +830,10 @@ public Blob getBlob(String columnLabel) throws SQLException {
 
   @Override
   public Clob getClob(String columnLabel) throws SQLException {
-    logger.debug("public Clob getClob(String columnLabel)", false);
+    logger.debug("public Clob getClob(String columnLabel)");
```

Comment:
why to remove "false" argument meant for masking?

Reply:
Thanks for spotting that! I think it was done in error when addressing merge conflicts locally after cherry-picking the commit to the main branch. I'll change that back.